vignettes/epidatr.Rmd

```diff
@@ -11,379 +11,224 @@ vignette: >
+requests for specific signals on specific dates and in select geographic
 regions.
 
-We recommend you register for an API key. While most endpoints are available
-without one, there are [limits on API usage for anonymous
-users](https://cmu-delphi.github.io/delphi-epidata/api/api_keys.html), including
-a rate limit. See `save_api_key()` for details on how to obtain an API key and
-set this package to use it.
 
-## Basic Usage
+## Setup
 
-Fetching some data from the Delphi Epidata API is simple. Suppose we are
-interested in the [`covidcast`
-endpoint](https://cmu-delphi.github.io/delphi-epidata/api/covidcast.html), which
-provides access to a range of data on COVID-19. Reviewing the endpoint
-documentation, we see that we need to specify a data source name, a signal name,
-a geographic level, a time resolution, and the location and times of interest.
+### Installation
 
-In this case, the `pub_covidcast()` function lets us specify these parameters for
-the endpoint and returns a tibble with the results:
+You can install the stable version of this package from CRAN:
 
-```{r}
-epidata <- pub_covidcast(
-  "fb-survey", "smoothed_cli", "state", "day", "pa",
-  epirange(20210105, 20210410)
-)
-epidata
+```R
+install.packages("epidatr")
+pak::pkg_install("epidatr")
+renv::install("epidatr")
 ```
 
-We can then easily plot the data using ggplot2:
+Or if you want the development version, install from GitHub:
 
-```{r, out.height="65%"}
-library(ggplot2)
-ggplot(epidata, aes(x = time_value, y = value)) +
-  geom_line() +
-  labs(
-    title = "Smoothed CLI from Facebook Survey",
-    subtitle = "PA, 2021",
-    x = "Date",
-    y = "CLI"
-  )
+```R
+# Install the dev version using `pak` or `remotes`
+pak::pkg_install("cmu-delphi/epidatr@dev")
+remotes::install_github("cmu-delphi/epidatr", ref = "dev")
+renv::install("cmu-delphi/epidatr@dev")
 ```
 
-The [Delphi Epidata API documentation](https://cmu-delphi.github.io/delphi-epidata/) has more information on the available endpoints and arguments. You can also use the `avail_endpoints()` function to get a table of endpoint functions:
+### API Keys
 
-```{r}
-avail_endpoints()
-```
+The Delphi API requires a (free) API key for full functionality. While most
+endpoints are available without one, there are
+[limits on API usage for anonymous users](https://cmu-delphi.github.io/delphi-epidata/api/api_keys.html),
+including a rate limit.
 
-Example queries with all the endpoint functions available in this package are given [below](#example-queries).
+To generate your key,
+[register for a pseudo-anonymous account](https://api.delphi.cmu.edu/epidata/admin/registration_form).
+See the `save_api_key()` function documentation for details on how to set up
+`epidatr` to use your API key.
 
-## Advanced Usage (Experimental)
+_Note_ that private endpoints (i.e. those prefixed with `pvt_`) require a
+separate key that needs to be passed as an argument. These endpoints require
+specific data use agreements to access.
```

Comment:
Need to include more info here. How can users get access to private endpoints? Do we have a page or contact email for that?

Reply:
Never found that information anywhere. @brookslogan do you know?

Reply:
I don't.  Pre-COVID I don't think there was even a procedure, but I'm not sure if there is one now.

Context: Pre-COVID I think most of these we probably didn't even have permission to re-share.  Many were for internal use (e.g., feeding into ILI-Nearby) + potentially sharing back/visualizing to the providers or maybe very limited pre-agreed-upon third parties (and even this last possibility I'm not sure ever actually happened).  One that I know of (NoroSTAT) was private just for testing purposes; it requires revival work now though so I'm not sure there's a point in broadening access.

Reply:
Will look into this and address separately.